crnk-gen/crnk-gen-openapi/src/main/java/io/crnk/gen/openapi/internal/schemas/TypeSchema.java

```diff
@@ -13,7 +13,7 @@
   public Schema schema() {
     Schema typeSchema = new StringSchema()
         .description("The JSON:API resource type (" + metaResource.getName() + ")");
```

Comment:
You may want to update the description to match.

Reply:
Fixed